pennylane_lightning/src/tests/Makefile

```diff
@@ -13,8 +13,7 @@
 # limitations under the License.
 CC = g++
 CFLAGS = -std=c++11 -g -Wall -fopenmp -fPIC -I/usr/include \
-	-I../include -I$(GOOGLETEST_DIR)/include \
-	-march=native
```

Comment:
Had problems locally on a new machine with this command. It should be safe to be removed as it is used for machine-specific optimized code, which is not particularly necessary when running unit tests.